lib/worker.coffee

```diff
@@ -59,10 +60,7 @@ registerBlockPositions = (doc) ->
       doc.id = "__asciidoctor-preview-#{doc.$object_id()}__"
```

Comment:
@mojavelinux I'm wondering if there's a more stable (unique) identifier than `object_id` ?

Reply:
Not at the moment, though this is not the first request for it. We'll need an upstream issue to discuss.